src/holofuel/components/PageDivider/PageDivider.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
As it will be referenced and used across at least three separate pages (ie: Inbox, Transaction History, Home), I think we should turn this into a design-system component.  It is simple, but there is not reason to repeat the code in each of these style-sheets.  (IMO, there are a few other components, like the nav badge, that would benefit from this too.  ....I also recognize that this is a design system question and we can talk it about it at more length when we take on that discussion.)